lms/djangoapps/support/static/support/jsx/errors_list.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types'
+
+class ShowErrors extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+
+    componentDidMount() {
+        window.scrollTo(0, 0)
+    }
+
+    render() {
+        return this.props.errorList.length > 0 &&
+                <div className="col-sm-12">
+                    <div className="alert alert-danger" role="alert">
+                        <strong>{gettext("Please fix following errors:")}</strong>
+                        <ul>
+                            {this.props.errorList.map(error =>
+                                <li>{error}</li>
+                            )
```

Comment:
Maybe collapse this line and the next into a single line?

Reply:
I will refactor this